corehq/apps/accounting/tasks.py

```diff
@@ -833,7 +833,9 @@ def calculate_web_users_in_all_billing_accounts(today=None):
     today = today or datetime.date.today()
     for account in BillingAccount.objects.all():
         record_date = today - relativedelta(days=1)
-        domains = account.get_domains()
+        # Get history of web user in account even for no longer activated domain
+        # thus we can get calculate old web user history
+        domains = account.get_domains(active_only=False)
```

Comment:
@AmitPhulera Hi Amit, what do you think of this change? This will get the inactive domain as well. I checked, in the `WebUser.ids_by_domain()`function. we only get active web user. So I guess the total number of active web user will still be accurate?

Reply:
Thanks @jingcheng16 for your question. I thought more about it, although the change should be  fine but I am curious to know the reasoning behind including inactive domains. Is it for the subscriptions that are deactivated in between the billing cycles?

Reply:
@AmitPhulera I include inactive domain for passing the test. Our test setup domain that only active for 15 months. When we calculate web users for a old date, then we have to include the inactive domain to pass the test.

Reply:
Okay. So we would not need this in production at all, right? Would it be possible to update the tests rather than updating the functionality here?

Reply:
Offlined with Amit. @AmitPhulera I've reverted my changes here and update the test case. Can you review it again?